superset-frontend/src/features/databases/CSVUploadModal/index.tsx

```diff
@@ -0,0 +1,805 @@
+                  name="skip_blank_lines"
+                >
+                  <Switch data-test="skipBlankLines" />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t(
+                    'Skip blank lines rather than interpreting them as Not A Number values',
+                  )}
+                </p>
+              </Col>
+            </Row>
+            <Row>
+              <Col span={24}>
+                <StyledFormItem
+                  label={t('Columns To Be Parsed as Dates')}
+                  name="column_dates"
+                >
+                  <Select
+                    ariaLabel={t('Choose columns to be parsed as dates')}
+                    mode="multiple"
+                    options={columnsToOptions()}
+                    allowClear
+                    allowNewOptions
+                  />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t(
+                    'A comma separated list of columns that should be parsed as dates',
+                  )}
+                </p>
+              </Col>
+            </Row>
+            <Row>
+              <Col span={24}>
+                <StyledFormItem label={t('Day First')} name="day_first">
+                  <Switch data-test="dayFirst" />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t('DD/MM format dates, international and European format')}
+                </p>
+              </Col>
+            </Row>
+            <Row>
+              <Col span={24}>
+                <StyledFormItem
+                  label={t('Decimal Character')}
+                  name="decimal_character"
+                >
+                  <Input type="text" defaultValue="." />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t('Character to interpret as decimal point')}
+                </p>
+              </Col>
+            </Row>
+            <Row>
+              <Col span={24}>
+                <StyledFormItem label={t('Null Values')} name="null_values">
+                  <Select
+                    mode="multiple"
+                    options={nullValuesOptions}
+                    allowClear
+                    allowNewOptions
+                  />
+                </StyledFormItem>
+                <p className="help-block">
+                  {t(
+                    'Json list of the values that should be treated as null. Examples: [""] for empty strings, ["None", "N/A"], ["nan", "null"]. Warning: Hive database supports only a single value',
+                  )}
+                </p>
+              </Col>
+            </Row>
+          </Collapse.Panel>
+          <Collapse.Panel
+            header={
+              <div>
+                <h4>{t('Columns')}</h4>
+                <p className="helper">{t('Adjust column settings.')}</p>
```

Comment:
Another small nit question on consistency in punctuation. Should we be using periods at the end of the helper text or no?

Reply:
I think so, followed what we have on database modal

Reply:
Suggested text change 
`Adjust column settings.` -> 
`Adjust column settings such as specifying the columns to read, how duplicates are handled, column data types, and more.`